src/NuGet.Core/NuGet.Build.Tasks/NuGetMessageTask.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace NuGet.Build.Tasks
+{
+    /// <summary>
+    /// A task that logs a message from the localized <see cref="Strings"/> resource.
+    /// </summary>
+    public sealed class NuGetMessageTask : Task
+    {
+        public NuGetMessageTask()
+            : base(Strings.ResourceManager)
+        {
+        }
+
+        public string[] Args { get; set; }
+
+        public string Importance { get; set; }
+
+        [Required]
+        public string Name { get; set; }
+
+        public override bool Execute()
+        {
+            MessageImportance messageImportance = MessageImportance.Normal;
+
+            if (!string.IsNullOrWhiteSpace(Importance) && !Enum.TryParse(Importance, ignoreCase: true, out messageImportance))
+            {
+                messageImportance = MessageImportance.Normal;
```

Comment:
`messageImportance` is already set to `Normal` in line no. 29. I am not sure if this is the right enum value to assign.

Reply:
If `Enum.TryParse` fails, it sets the out variable to `0` which is `High`

https://github.com/microsoft/msbuild/blob/e70a3159d64f9ed6ec3b60253ef863fa883a99b1/src/Framework/BuildMessageEventArgs.cs#L15